string effectorName
float32[] targetCoordinate
---
